### PR TITLE
Fix style problems

### PR DESCRIPTION
**Pull request by tnotheis**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

File: Applications/AdminCli/src/AdminCli/Commands/Clients/CreateClientCommand.cs [hunk 1]
```diff
@@ -53,9 +53,10 @@ private async Task CreateClient(string dbProvider, string dbConnectionString, st
     {
         var mediator = _serviceLocator.GetService<IMediator>(dbProvider, dbConnectionString);
 
-        var response = await mediator.Send(new Modules.Devices.Application.Clients.Commands.CreateClient.CreateClientCommand(clientId, displayName, clientSecret, defaultTier, maxIdentities), CancellationToken.None);
+        var response = await mediator.Send(new Modules.Devices.Application.Clients.Commands.CreateClient.CreateClientCommand(clientId, displayName, clientSecret, defaultTier, maxIdentities),
+            CancellationToken.None);
 
         Console.WriteLine(JsonSerializer.Serialize(response, JSON_SERIALIZER_OPTIONS));
-        Console.WriteLine("Please note the secret since you cannot obtain it later.");
+        Console.WriteLine(@"Please note the secret since you cannot obtain it later.");
```
Comment (Dannyps) [inline Applications/AdminCli/src/AdminCli/Commands/Clients/CreateClientCommand.cs:60]: Why?

**Review by Dannyps — COMMENTED** [review]
It seems that many of the changes here could be lint-checked. Would it be possible to change the linter's configuration to enforce this particular style?

**Comment by tnotheis** [comment]
> It seems that many of the changes here could be lint-checked. Would it be possible to change the linter's configuration to enforce this particular style?

It surely would. And I would love to do that one day. But I'm pretty sure this would take days to configure. :(

File: Applications/AdminCli/src/AdminCli/Commands/Clients/CreateClientCommand.cs (hunk 1 above)
Comment (tnotheis) [reply Applications/AdminCli/src/AdminCli/Commands/Clients/CreateClientCommand.cs:60]: To be honest: I have no idea. 😆 Rider complained about it ("Localizable interpolated string") and I didn't want to start a discussion about it. 😜

**Review by Dannyps — APPROVED** [review]

**Pull request merged**